src/lua/fiber.lua

```diff
@@ -24,19 +25,24 @@ Sets the default value for the max fiber slice. The old value is infinity
 https://tarantool.io/compat/fiber_slice_default
 ]]
 
+local max_slice_default
+if tarantool.build.asan then
+    max_slice_default = TIMEOUT_INFINITY
```

Comment:
Please add a comment explaining why this is required.

Reply:
Done. 

```diff
--- a/src/lua/fiber.lua
+++ b/src/lua/fiber.lua
@@ -25,6 +25,8 @@ Sets the default value for the max fiber slice. The old value is in>
 https://tarantool.io/compat/fiber_slice_default
 ]]

+-- ASAN build is slow. Turn slice check off to suppress noisy failures
+-- on exeeding slice limit in this case.
 local max_slice_default
 if tarantool.build.asan then
     max_slice_default = TIMEOUT_INFINITY
```

Reply:
I'd added the same comment to `fiber.c` as well.

Reply:
Done.

```diff
diff --git a/src/lib/core/fiber.c b/src/lib/core/fiber.c
index 2fda2eec9..611abaa11 100644
--- a/src/lib/core/fiber.c
+++ b/src/lib/core/fiber.c
@@ -50,6 +50,11 @@
 extern void cord_on_yield(void);

 static struct fiber_slice zero_slice = {.warn = 0.0, .err = 0.0};
+
+/**
+ * ASAN build is slow. Turn slice check off to suppress noisy failures
+ * on exceeding slice limit in this case.
+ */
 #if ENABLE_ASAN
 static struct fiber_slice default_slice = {
        .warn = TIMEOUT_INFINITY,
@@ -58,6 +63,7 @@ static struct fiber_slice default_slice = {
 #else
 static struct fiber_slice default_slice = {.warn = 0.5, .err = 1.0};
 #endif
+
 /** Number of cord-threads still running right now. */
 static int cord_count = 0;
```